### PR TITLE
Move riscv64 to git

### PR DESCRIPTION
**Pull request by pevik**

### CODE REVIEW

File: job_groups/opensuse_tumbleweed_riscv64.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  opensuse-Tumbleweed-JeOS-for-RISCV-riscv64:
+    distri: opensuse
+    flavor: JeOS-for-RISCV
+    version: Tumbleweed
+scenarios:
+  riscv64:
+    opensuse-Tumbleweed-JeOS-for-RISCV-riscv64:
+      - jeos:
+          settings:
+            EXCLUDE_MODULES: diskusage,libzypp_config
```
Comment (Vogtinator) [inline job_groups/opensuse_tumbleweed_riscv64.yaml:27]: I just changed this on o3 (https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/19899 got merged):

```suggestion
            EXCLUDE_MODULES: diskusage,libzypp_config
```

**Review by Vogtinator — APPROVED** [review]

**Comment by Vogtinator** [comment]
Looks like the requested change wasn't included?

**Comment by Vogtinator** [comment]
> Looks like the requested change wasn't included?

@pevik ping

**Comment by pevik** [comment]
I'm sorry, I'll create a new PR.

File: job_groups/opensuse_tumbleweed_riscv64.yaml (hunk 1 above)
Comment (pevik) [reply job_groups/opensuse_tumbleweed_riscv64.yaml:27]: https://github.com/os-autoinst/opensuse-jobgroups/pull/501

**Pull request merged**